### DISCUSSION TITLE
API UseCase 문서 관리

### DISCUSSION THREAD

**Discussion by Dae-Hwa** [Sc0up/scoup-backend#88] [Documents]
구글 시트로 되어있으니 개인적으로 보기가 살짝 불편해서 노션에서 한 번 해봤어요

https://www.notion.so/3b0c24a75fc9453995a4ee6f2a68ddc5?v=dc737d0d26e344f2b2b181f82e647fba

혹시 구글시트가 더 좋으시다면 원래 쓰던거 계속 쓰도록 합시다 ㅋㅋ

노션이 더 좋으면 옮기는건 제가 해도 상관없을듯 합니다

**Comment by janeljs** [comment]
아예 인수조건을 페이지 안으로 넣으셨군요. 저렇게하면 작업한 사람 외에는 뭔가 잘못된 변경이 생겨도 알아차리기 더 힘들지 않을까요? 케이스 별로 인수조건 비교할 때도 들어가서 비교해야 하구요!

**Comment by Dae-Hwa** [comment]
노션에서 관리
